django_forest/utils/cors.py

```diff
@@ -16,15 +16,10 @@ def set_cors():
         common_middleware_index = 0
     settings.MIDDLEWARE.insert(common_middleware_index, 'corsheaders.middleware.CorsMiddleware')
 
-    cors_allowed_origins = ['http://localhost:4200']
-    cors_allowed_origins.extend(get_list_setting('CORS_ALLOWED_ORIGINS'))
-
     cors_allowed_origins_regexes = [r'.*\.forestadmin\.com.*']
     cors_allowed_origins_regexes.extend(get_list_setting('CORS_ALLOWED_ORIGIN_REGEXES'))
 
-    settings.CORS_ALLOWED_ORIGINS = cors_allowed_origins
     settings.CORS_ALLOWED_ORIGIN_REGEXES = cors_allowed_origins_regexes
-    settings.CORS_URLS_REGEX = r'^/forest(/.*)?$'  # restrict it to /forest
```

Comment:
You will need this line.

Reply:
I need it for what ?

Reply:
It's just limit the urls for which the cors headers are sent. 
If  I have some routes that need these headers, it's won't work.
https://github.com/adamchainz/django-cors-headers#cors_urls_regex-str--patternstr

Reply:
Please look at the node.js and ruby SDK